packages/repository/README.md

```diff
@@ -168,7 +168,13 @@ coercion. The following types are supported out of box.
 - ArrayType
 - UnionType
 
-# Use Repository
+## Installation
+
+```
+$ npm install --save @loopback/repository
+```
+
+## Basic use
```

Comment:
I see, sorry for getting confused about your question, @loay.

> I would create a new section and call it `Concept` and move it under `Related Source`, if everybody is ok with that.

Let's do that now, to get this pull request landed 👍